### ISSUE TITLE
Grafana 8.0 compatibility

### ISSUE THREAD

**Issue by tilsche** [metricq/grafana-metricq-datasource#6]
It seems that the current version is not compatible with Grafana 8.0.

I wonder if this depends on #4 ?

**Comment by kinnarr** [comment]
This is probably caused by grafana/grafana#34364. To allow unsigned plugins, see https://grafana.com/docs/grafana/latest/administration/configuration/#allow_loading_unsigned_plugins

Signing will be addressed in a new issue.
